Source/embeddings/Internal/Embedding.ts

```diff
@@ -57,7 +57,7 @@ export class Embedding<T> implements IEmbedding<T> {
 
     /** @inheritdoc */
     compare(receivedState: T, currentState: T, context: EmbeddingContext) {
```

Comment:
```suggestion
    update(receivedState: T, currentState: T, context: EmbeddingContext) {
```

We should rename this also to `update` like we did in the [C# SDK](https://github.com/dolittle/DotNET.SDK/blob/17f6ceb273f6473a255c491bee8ec7d29ea7079b/Source/Embeddings/Internal/Embedding.cs#L61).

We should also do a try catch here (and in the delete and on methods too) and throw a more meaningful exception (like I said in my comment from earlier review). We can write them together too.